recipe/meta.yaml

```diff
@@ -58,4 +59,5 @@ extra:
     - bgruening
     - daler
     - jdblischak
-    - bsennblad
```

Comment:
@ConorIA Could you please add back bsennblad as a maintainer?

Reply:
@jdblischak, sorry. Fixed. The skeleton generator added cbrueffer as well. @cbrueffer, please let me know if I should remove you.

Reply:
@ConorIA That's a good instinct, and in general you don't want to add maintainers without their explicit approval. However, in this case you don't need to ask because @cbrueffer is listed as a maintainer in the helper script:

https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml